tests/spectron/alwaysOnTop.spectron.js

```diff
@@ -1,190 +1,65 @@
-                    done();
-                } else {
-                    expect(isAlwaysOnTop).toBeFalsy();
-                    done();
-                }
-                done();
-            });
-        }).catch((err) => {
-            done.fail(new Error(`alwaysOnTop failed in readConfig with error: ${err}`));
-        });
-    });
-
-    it('should toggle the always on top property to true', (done) => {
-        if (isMac) {
-            robot.setMouseDelay(200);
-            robot.moveMouse(190, 0);
-            robot.mouseClick();
-            // Key tap 7 times as "Always on Top" is in the
-            // 7th position under view menu item
-            for (let i = 0; i < 7; i++) {
-                robot.keyTap('down');
             }
-            robot.keyTap('enter');
-            done();
-        } else {
-            app.browserWindow.getBounds().then((bounds) => {
-                app.browserWindow.focus();
-                robot.setMouseDelay(200);
-                app.browserWindow.restore().then(() => {
-                    let x = bounds.x + 95;
-                    let y = bounds.y + 35;
-
-                    robot.moveMouseSmooth(x, y);
-                    robot.mouseClick();
-                    // Key tap 4 times as "Always on Top" is in the
-                    // 4th position under window menu item
-                    for (let i = 0; i < 4; i++) {
-                        robot.keyTap('down');
-                    }
-                    robot.keyTap('enter');
-                    done();
-                });
-            }).catch((err) => {
-                done.fail(new Error(`alwaysOnTop failed in getBounds with error: ${err}`));
-            });
-        }
+        } catch (err) {
+            done.fail(new Error(`Failed at post-condition: ${err}`));
+        };
     });
 
-    it('should check is always on top to be true', (done) => {
-        if (!mIsAlwaysOnTop) {
-            return app.browserWindow.isAlwaysOnTop().then((isAlwaysOnTop) => {
-                expect(isAlwaysOnTop).toBeTruthy();
-                done();
-            }).catch((err) => {
-                done.fail(new Error(`alwaysOnTop failed in isAlwaysOnTop with error: ${err}`));
-            });
-        } else {
-            return app.browserWindow.isAlwaysOnTop().then((isAlwaysOnTop) => {
-                expect(isAlwaysOnTop).toBeFalsy();
-                done();
-            }).catch((err) => {
-                done.fail(new Error(`alwaysOnTop failed in isAlwaysOnTop with error: ${err}`));
-            });
-        }
+    /**
+     * Verify Always on Top options when multiple applications are opened
+     * TC-ID: 2898431
+     * Cover scenarios in AVT-990
+     */
+    it('Verify Always on Top options when multiple applications are opened', async (done) => {
+        try {
+            await webActions.minimizeWindows();
+            await windowActions.openAppInMaximize("C:\\Windows\\notepad.exe");
```

Comment:
Should move `openAppInMaximize` method to other.

Reply:
Updated